ops/docker/deployment-utils/Dockerfile

```diff
@@ -33,3 +33,10 @@ COPY --from=base /root/.foundry/bin/forge /usr/local/bin/forge
 COPY --from=base /root/.foundry/bin/cast /usr/local/bin/cast
 COPY --from=base /root/.foundry/bin/anvil /usr/local/bin/anvil
 COPY --from=go-base /go/bin/dasel /usr/local/bin/dasel
+
+# Install geth
+RUN curl -L https://gethstore.blob.core.windows.net/builds/geth-linux-amd64-1.15.11-36b2371c.tar.gz \
```

Comment:
Could we check the hash of the file we download?

Reply:
Good idea, done!